test/common/index.js

```diff
@@ -102,6 +102,43 @@ function expectsError (validator, exact) {
   }, exact)
 }
 
+if (typeof AbortSignal.timeout !== 'function') {
```

Comment:
will this be required in consumer code too? Would it be safer to use a AbortSignal ponyfill?

Reply:
no, it is not needed, these adjustments are just so the tap output will match across node versions

Reply:
Great, thank you!

```suggestion
// Ensure test output is consistent across node versions
if (typeof AbortSignal.timeout !== 'function') {
```

Reply:
`AbortSignal.timeout` doesn't exist on Node.js 14.x, so all the tests that use it fails on that version. It's not required though, `test/node-core-test.js` makes sure that the library still works when it's not polyfilled.

The section below, that overrides `AbortSignal.abort` and `AbortController.prototype.abort` is indeed there just for the TAP output to match, and same, we have `test/node-core-test.js` to ensure it's not necessary for the lib to work.

Reply:
I'm going to land this and open a follow-up PR that adds the comments.